mode/type/cluster/repository/api/src/main/java/org/apache/shardingsphere/mode/repository/cluster/lock/holder/DistributedLockHolder.java

```diff
@@ -33,14 +35,14 @@ public final class DistributedLockHolder {
     
     private final TypedProperties<?> props;
     
-    private final Map<String, DistributedLock> locks;
+    private final ConcurrentHashMap<String, DistributedLock> locks;
```

Comment:
Should we use `ConcurrentMap` or `Map` here?

Reply:
sure